src/components/navbar/index.js

```diff
@@ -1,18 +1,54 @@
-import React from 'react'
+import React, { PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { Link } from 'react-router'
+
 import Image from '../image'
 
 import styles from './styles.css'
 
-const logo = require('../../../assets/img/logo.png')
+import logo from '../../../assets/img/logo.png'
+
+const logoutImg = require('../../../assets/img/logout.png')
```

Comment:
we should use `import`, we may have to set up webpack to handle this kind of file